lms/resources/_js_config/__init__.py

```diff
@@ -226,6 +226,7 @@ def enable_content_item_selection_mode(self, form_action, form_fields):
                 },
             }
         )
+        self._add_deeplinking_api()
```

Comment:
Done as separate call instead of directly assign to `_config['deeplinkingAPI']` to allow `_add_deeplinking_api` to leave the config untouched (when I split this and do it only for 1.3)